fluent-syntax/src/parser.js

```diff
@@ -416,34 +415,37 @@ export default class FluentParser {
   getValue(ps) {
     if (ps.currentIs("{")) {
       ps.peek();
-      ps.peekInlineWS();
-      if (ps.isPeekNextLineVariantStart()) {
+      ps.peekBlank();
+      if (ps.currentPeek() === "*" || ps.currentPeek() === "[") {
+        ps.resetPeek();
         return this.getVariantList(ps);
       }
+
+      ps.resetPeek();
     }
 
     return this.getPattern(ps);
   }
 
   getVariantList(ps) {
     ps.expectChar("{");
-    ps.skipInlineWS();
+    ps.skipBlank();
```

Comment:
I think this needs to ensure at least one line break before moving on to parsing variants. Same as you do in `getExpression`.